Dockerfile

```diff
@@ -6,6 +6,7 @@ FROM golang:1.17-alpine as builder
 WORKDIR /app
 COPY go.mod go.sum ./
 RUN go mod download
+COPY cmd/backup/notifications.tmpl ./cmd/backup/notifications.tmpl
```

Comment:
I think we could also just do

```
COPY cmd/backup ./cmd/backup/
```

now. Which will also create a good incentive for me to start splitting this "little file" that has grown to be 1000+LOC by now into chunks at some point :)